### ISSUE TITLE
Intersphinx mapping stopped working

### ISSUE THREAD

**Issue by reszelaz** [tango-controls/pytango#351]
I don't know if this is an issue with RTD or PyTango, but I realized that the intersphinx mapping to PyTango documentation that we use in Sardana gives errors in the last days:

```
WARNING: intersphinx inventory 'http://pytango.readthedocs.io/en/stable/objects.inv' not fetchable due to <class 'urllib.error.HTTPError'>: HTTP Error 403: Forbidden
```

**Comment by ajoubertza** [comment]
Hi @reszelaz

What is that file for?

Anyway, I didn't change anything on TRD other than new release (so stable branch updated).  I notice that using HTTPS results in a reasonable file, while just HTTP does not.  Maybe RTD changed their security policy?

```
$ curl http://pytango.readthedocs.io/en/stable/objects.inv -o http-obj.inv
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   154    0   154    0     0   3142      0 --:--:-- --:--:-- --:--:--  3142

$ curl https://pytango.readthedocs.io/en/stable/objects.inv -o https-obj.inv
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  5468  100  5468    0     0  88193      0 --:--:-- --:--:-- --:--:-- 88193

$ ll *.inv
-rw-r--r--  1 aj  aj   154 May  5 21:49 http-obj.inv
-rw-r--r--  1 aj  aj  5468 May  5 21:49 https-obj.inv

$ head http-obj.inv 
<html>
<head><title>302 Found</title></head>
<body bgcolor="white">
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>

$ head https-obj.inv 
# Sphinx inventory version 2
# Project: PyTango
# Version: 9.3
# The remainder of this file is compressed using zlib.
xڵ?Ks?8????1?m????ܼ???X?W?r??
...
``` 

I also tried with earlier versions in the URL and it is the same - only HTTPS gives the large file.

**Comment by reszelaz** [comment]
Hi Anton,
Thanks for investigating it!
My first idea was also the https, but I tested it and it fails the same way.
Our intersphinx mapping is defined in [`conf.py`](https://github.com/sardana-org/sardana/blob/d866e4c7fe169aa8271970f87fa5719368b63793/doc/source/conf.py#L291).
Good to know that there were no changes in the last release. Now I tested it in a venv with the latest sphinx from pip and it works. I will update our docker image that we use in CI to build docs with the newest sphinx.
